src/video/camerasource.cpp

```diff
@@ -214,14 +233,18 @@ void CameraSource::unsubscribe()
     }
     else
     {
+        //qDebug() << "second block";
         device->close();
         biglock = false;
     }
+    subscriptions--;
```

Comment:
In common case prefix is faster than postfix:
http://www.viva64.com/en/d/0165/print/
https://programmers.stackexchange.com/questions/59880/avoid-postfix-increment-operator
http://www.viva64.com/en/b/0093/
With std::atomic_int is difficult to say slower to be postfix or not, but prefix certainly not slower.